src/Formatter/CallableFormatter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace League\BooBoo\Formatter;
+
+use Exception;
```

Comment:
In the League it's recommended to import everything. Doesn't really matter though.